tests/testthat/testdata/test_configs/config_invalid_set_filter_task_name.yaml

```diff
@@ -0,0 +1,97 @@
+schema_version: https://raw.githubusercontent.com/hubverse-org/hubPredEvalsData/main/inst/schema/v1.0.0/config_schema.json
+targets:
+- target_id: wk inc flu hosp
+  metrics:
+  - wis
+  - ae_median
+  - interval_coverage_50
+  - interval_coverage_95
+  disaggregate_by:
+  - location
+  - reference_date
+  - horizon
+  - target_end_date
+- target_id: wk flu hosp rate category
+  metrics:
+  - log_score
+  - rps
+  disaggregate_by:
+  - location
+  - reference_date
+  - horizon
+  - target_end_date
+eval_sets:
+- eval_set_name: Full season
+  round_filters:
+    min: '2023-01-21'
+- eval_set_name: Last 4 weeks
+  round_filters:
+    min: '2023-01-21'
+    n_last: 4
+- eval_set_name: set with invalid task id variable name
+  round_filters:
+    min: '2023-01-21'
+  task_filters:
+    location: ["01"]
+    horizon: [1, 2]
+    not_a_real_task_id: 4
```

Comment:
this line results in an error because `not_a_real_task_id` is not the name of a task id variable for the hub.